snuba/utils/bucket_timer.py

```diff
@@ -88,6 +88,7 @@ def get_projects_exceeding_limit(self) -> List[int]:
         now = datetime.now()
         self.__trim_expired_buckets(now)
         project_groups: dict[int, timedelta] = defaultdict(lambda: timedelta(seconds=0))
+        logger.debug("self.buckets_size: %s" % len(self.buckets))
```

Comment:
is this all that you want to instrument? It takes a while to deploy something

Reply:
no I'm still drafting